canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -374,6 +389,25 @@ class ToolTip extends React.Component {
 			tipClass += " " + this.props.className;
 		}
 
+		let link = null;
+		if (this.state.isTooltipVisible && this.props.tooltipLinkHandler && this.props.link && this.props.propertyId) {
+			this.linkInformation = this.props.tooltipLinkHandler(
```

Comment:
Why do we need to set this as a class variable?  We can bind it on `tooltipLinkOnClick`.